### DISCUSSION TITLE
Open saved/offline webpage if internet not available.

### DISCUSSION THREAD

**Discussion by servoo** [plateaukao/einkbro#330] [Ideas]
In NeoBrowser, its possible to open the offline version of a page by clicking a link in a PDF when internet is not available, but the page looks messed up. AFAIK, this is not possible in EinkBro. This would make so much sense to add as a feature.

I realize you can use Pocket to save. But this does not take care of opening a link from a pdf, and directly being taken to the offline version of a page.

**Comment by plateaukao** [comment]
@servoo 
I don't quite get your idea.If you click a link in a pdf, and it's offline, how can EinkBro know that in advance and cache it or save it somewhere in advance?

Could you send me a pdf link, and upload a screenshot of what neoBrowser shows when a link in pdf is clicked? I don't think neoBrowser can really do it either.

**Reply by servoo** [reply]
First and foremost, thank you for your reply.

I will provide pictures/gifs later. Sorry for the lack of info in my request.

For EinkBro, as you stated, you need to download the webpage beforehand. So in that sense, it is cached (in the download folder). The next time you click the same link in the pdf, the cached page will open. This makes it very convenient to read webpages without internet, when clicking links in pdfs.
